etcdctlv3/README.md

```diff
@@ -323,3 +323,19 @@ Simple reply
 
 [etcdrpc]: ../etcdserver/etcdserverpb/rpc.proto
 [storagerpc]: ../storage/storagepb/kv.proto
+
+## Compatibility Support
+
+etcdctl is still in its early stage. We try out best to ensure fully compatible releases, however we might break compatibility to fix bugs or improve commands. If we intend to release a version of etcdctl with backward incompatibilities, we will notice users before releases and talk about how to upgrade smoothly.
```

Comment:
`we will provide notice prior to release and have instructions on how to upgrade`